lib/stellar_core_commander/local_process.rb

```diff
@@ -61,7 +61,7 @@ def history_dir
     Contract None => Any
     def setup!
       write_config
-      create_database unless @keep_database
+      create_database unless @keep_database or is_sqlite
```

Comment:
maybe it would be better to force a value for `keep_database` instead?

Reply:
that seems confusing. using sqlite does not necessarily imply keeping the database.